common/getPagePathnameInLanguage.js

```diff
@@ -1,6 +1,5 @@
 export const getPathName = (path) => {
-  const [, , , pathname] = path.match(/^\/(.*?)(\/(.*))?$/)
-
+  const pathname = path.replace(/^\/(?<levelOne>.*?)(?<levelTwo>\/(?<subLevelTwo>.*))?$/, '$3')
```

Comment:
Давай вот такие штуки писать более явно: 
```suggestion
  const regexp = /^\/(?<levelOne>.*?)(?<levelTwo>\/(?<subLevelTwo>.*))?$/
  const pathname = regexp.exec(path).groups.subLevelTwo
```

Reply:
Проверь другие regexp

Reply:
попробуй удалить  `node_modules` и переустановить пакеты